variables.tf

```diff
@@ -60,3 +52,8 @@ variable "table_labels" {
   description = "Key value pairs in a map for table labels"
   type        = "map"
 }
+
+variable "tables" {
+  description = "A list of table IDs that will be created on the single dataset"
```

Comment:
Based on the usage of this variable in `main.tf`, this is actually a list of maps which include a `table_id` and a `schema`. This expected structure needs to be documented in the description.

Reply:
will be changed in next commit